src/osd/scrubber/scrub_machine.h

```diff
@@ -154,10 +154,11 @@ struct NotActive : sc::state<NotActive, ScrubMachine> {
   using reactions = mpl::list<sc::custom_reaction<StartScrub>,
 			      // a scrubbing that was initiated at recovery completion,
 			      // and requires no resource reservations:
```

Comment:
Can you remove line 156 ('and requires...')? it's no longer correct.
(not exactly part of this PR, but pretty much related...)